src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -106,6 +106,22 @@ public function getName()
 
     public function setRequirements($requirements)
     {
+        if (isset($requirements['_method'])) {
+            if (0 === count($this->methods)) {
+                $this->methods = explode('|', $requirements['_method']);
+            }
+
+            @trigger_error('The "_method" requirement is deprecated since version 2.2 and will be removed in 3.0. Use the setMethods() method instead and use the "methods" option instead of the "_method" key of the "requirements" option in the route definition.', E_USER_DEPRECATED);
```

Comment:
I think `Use the "methods" option instead.` should be enough. No user manually calls setMethods on the annotation class and the sentence is too confusing.